### PR TITLE
Added FeaturedRoomCardHosts (#1324)

### PR DESCRIPTION
**Pull request by BronzW**
I've first of all added the FeaturedRoomCardHosts component (+story) in #1324  and I've made a temporary ``md`` until the implementation of it in #1390. I've also made it so the bigger the icons in ``MultipleUsers`` the further apart they are, for visibilty. When they are very small you don't really care that they are close together, but when they are big you almost cannot see the picture behind.

### CODE REVIEW

**Comment by goldyydev** [comment]
FeaturedRoomCardAvatars does not seem to match the [Figma ](https://www.figma.com/file/CS01VVLR7ArQl0afYFkNj3/Web-App?node-id=4300%3A2338)design. 

Also, you have some conflicts

**Comment by BronzW** [comment]
Yeah saw that, I'm going to fix that

**Comment by BronzW** [comment]
Fixed it!

**Comment by BronzW** [comment]
I'm done now.

**Comment by ajmnz** [comment]
The square border is just for delimitation purposes, it shouldn't be there

**Comment by BronzW** [comment]
There we go @ajmnz

**Review by HarrisonMayotte — APPROVED** [review]

**Comment by goldyydev** [comment]
The text does not appear to be centered, that cold cause us issues in the future.
image

**Comment by BronzW** [comment]
@goldyydev Fixed it

**Comment by BronzW** [comment]
Anyone? I'm feeling lonely and scared

File: kibbeh/src/ui/mobile/FeaturedRoomCardHosts.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <div className="flex-row px-6 py-6 align-middle">
+        <FeaturedRoomCardAvatars avatars={avatars} />
+        <div className="flex-col pl-4 justify-center">
+            <p className="text-primary-300">Hosted by</p>
+            <p className="text-primary-100">{ names.join(', ') }</p>
+        </div>
+    </div>
+  );
+};
```
Comment (goldyydev) [inline kibbeh/src/ui/mobile/FeaturedRoomCardHosts.tsx]:
```suggestion
}; 

```

File: kibbeh/src/ui/mobile/FeaturedRoomCardAvatars.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+              width: 50,
+              overflow: "hidden",
+            }}
+          >
+            <SingleUser src={s} size="md" />
+          </span>
+        ))}
+    </div>
+  );
+};
```
Comment (goldyydev) [inline kibbeh/src/ui/mobile/FeaturedRoomCardAvatars.tsx]:
```suggestion
}; 

```

File: kibbeh/src/stories/mobile/FeaturedRoomCardHosts.story.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+};
+
+const One: Story<FeaturedRoomCardHostsProps> = () => {
+  return (
+    <FeaturedRoomCardHosts avatars={[src]} names={['Marcus Bloch']} />
+  );
+};
+
+export const Main = Hosts.bind({});
+export const OneHost = One.bind({})
```
Comment (goldyydev) [inline kibbeh/src/stories/mobile/FeaturedRoomCardHosts.story.tsx]:
```suggestion
export const OneHost = One.bind({});

```

File: kibbeh/src/stories/mobile/FeaturedRoomCardAvatars.story.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+};
+
+const One: Story<FeaturedRoomCardAvatarsProps> = () => {
+  return (
+    <FeaturedRoomCardAvatars avatars={[src]} />
+  );
+}
+
+export const Main = Avatars.bind({});
+export const OneAvatar = One.bind({})
```
Comment (goldyydev) [inline kibbeh/src/stories/mobile/FeaturedRoomCardAvatars.story.tsx]:
```suggestion
export const OneAvatar = One.bind({});
```

**Review by goldyydev — CHANGES_REQUESTED** [review]

**Pull request merged**